prime-router/metadata/schemas/iPatientCare/ReddyFMC-LA-covid-19.schema

```diff
@@ -20,8 +20,18 @@ elements:
       mapper: concat(patient_id, filler_order_id)
 
     - name: test_kit_name_id
+      type: CODE
+      csvFields: [{name: TestName, format: $alt}]
+      altValues:
+        - code: 10811877011290
+          display: "SARS-CoV-2 (COVID-19) Ag"
+        - code: 10811877011290
+          display: "     SARS-CoV-2 (COVID-19) Ag"
```

Comment:
Are these estar spaces needed?

Reply:
Yes, some of the entries are offset with spaces